src/NotificationTargetCommonITILObject.php

```diff
@@ -1164,8 +1166,8 @@ public function getDataForObject(CommonDBTM $item, array $options, $simple = fal
                     $users[] = $user_tmp->getName();
 
                     $tmp = [];
```

Comment:
You could probably use `$tmp = self::getActorData($user_tmp, CommonITILActor::REQUESTER, 'author');`.
It is not a problem if there are some more tags, you just have to ensure that all existing tags are kept.